src/main/java/com/github/jasminb/jsonapi/ResourceConverter.java

```diff
@@ -346,13 +346,23 @@ private void handleRelationships(JsonNode source, Object object, Map<String, Obj
 					if (resolveRelationship && resolver != null && relationship.has(LINKS)) {
 						JsonNode self = relationship.get(LINKS).get(SELF);
```

Comment:
It would be great if there was a method that consumes `link` node and returns either its string representation if present or `href` attribute in case of an object.

This than could be reused for handling self/related use-cases mentioned in #21